recipes-bsp/ti-pruss-firmware/ti-pruss-firmware_08.02.00.002.bb

```diff
@@ -22,12 +22,12 @@ SRC_URI = " \
     ${@get_prueth_fw(d, 'am65x-pru1-prueth-fw.elf', '56cb71d205810a3609a5dff950fa7d7ea6b383d9d2fb6b1f2116ba58527f2f7f')} \
     ${@get_prueth_fw(d, 'am65x-rtu0-prueth-fw.elf', 'f01f58cdfad61652205c7ea6ed41ef0a759f63f02551447d940f919be691543f')} \
     ${@get_prueth_fw(d, 'am65x-rtu1-prueth-fw.elf', '3b905819ef3a7c4daa560106f3bba62cace55380c5ce40a22f7de232c0083052')} \
-    ${@get_prueth_fw(d, 'am65x-sr2-pru0-prueth-fw.elf', '21526f5c2a6a78f31e0e060867e41793b2470457ec0ebcff0ce38e3dba8a3d7f')} \
-    ${@get_prueth_fw(d, 'am65x-sr2-pru1-prueth-fw.elf', 'b19df885af9b2b5724a4380e1498ad83911bcf238a1b9968e9be3c33c1956f15')} \
-    ${@get_prueth_fw(d, 'am65x-sr2-rtu0-prueth-fw.elf', '74aa86d4c9e9e699f683fac0bdea5a78dee3a3dbd4ce0b903ed91055f043e059')} \
-    ${@get_prueth_fw(d, 'am65x-sr2-rtu1-prueth-fw.elf', '179495fe4f77782d74f99f67ab9aaed6eb66405c9b87f47f346cdb5f03f95d26')} \
-    ${@get_prueth_fw(d, 'am65x-sr2-txpru0-prueth-fw.elf', 'd6c2eee72555c86fa4fe6e20fafd54dcee5403196856aed6750374e8c9a0d2c3')} \
-    ${@get_prueth_fw(d, 'am65x-sr2-txpru1-prueth-fw.elf', '8923ec1b1e82dd794440957c5149f370e239c8b782666aba7911fdc76c419d95')} \
+    ${@get_prueth_fw(d, 'am65x-sr2-pru0-prueth-fw.elf', '8b709dbe96b5ae793e4865f24708c973ea2343df1c20e2199ba139868821eb93')} \
+    ${@get_prueth_fw(d, 'am65x-sr2-pru1-prueth-fw.elf', 'aacc9f2359fbd0dcb251cd8fd48275257a95075b640872192099552fafef4678')} \
+    ${@get_prueth_fw(d, 'am65x-sr2-rtu0-prueth-fw.elf', 'fb154656f7cf2dd398a35968dfeaedda1fd57e916ee82b8d6fe58ad4d018e53b')} \
+    ${@get_prueth_fw(d, 'am65x-sr2-rtu1-prueth-fw.elf', 'fc29c8f5e34234e96dcdda3a2f8628dec61c2c150e5e2896075b13a2d859acd6')} \
+    ${@get_prueth_fw(d, 'am65x-sr2-txpru0-prueth-fw.elf', '96e1dd06feb3f05915dc61fb1b11b27674484749f9881ef02ebb48ac07f1386e')} \
+    ${@get_prueth_fw(d, 'am65x-sr2-txpru1-prueth-fw.elf', '389ff8bdee84d66f0b1a8e02c2b8517b0b61a263fb65c18761e40eb83e09e9d2')} \
```

Comment:
The new firmware works with the current driver?

Reply:
Have not try to use this way.

Reply:
OK, patch ordering as shown by github always confuses me: this update comes after the driver update. I then assume you tested that the new driver already works with the older firmware.

Reply:
HI jan. 

Cause I have not tested new icssg whether it can work with older prufw, So I use one commit to contain new icssg driver and the prufw.

Reply:
@jan-kiszka  pls help to review this MR.  thanks a lot